cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/MapReduceContainerLauncher.java

```diff
@@ -40,7 +40,7 @@
    * Launches the given main class. The main class will be loaded through the {@link MapReduceClassLoader}.
    *
    * @param mainClassName the main class to launch
-   * @param args arguments for the main class
+   * @param args          arguments for the main class
```

Comment:
nit - seems like extra space was brought into the javadoc during conflict resolution